app/commands/user/reputation_token/award_for_pull_request.rb

```diff
@@ -34,19 +35,17 @@ def award_reputation_to_author
       def award_reputation_to_reviewers
         return unless just_closed?
 
-        reviews = octokit_client.pull_request_reviews(repo, pr_id)
-        reviewer_usernames = reviews.map { |reviewer| reviewer[:user][:login] }
+        reviewer_usernames = reviews.map { |reviewer| reviewer[:user][:login] }.uniq
```

Comment:
I added a `.uniq` at the end don't process the same reviewer more than once (there is no functional change though).